main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const AutoRunFile = ".autorun"
-const Version = "1.22.4dev"
+const Version = "1.22.5dev"
```

Comment:
Okay, fixing. I also wrote in my post "Changed version number, idk if you're okay with that.".